packages/components/src/internal/components/entities/DataTypeSelector.tsx

```diff
@@ -0,0 +1,259 @@
+                const nounPlural = entityDataType?.nounPlural?.toLowerCase() ?? 'samples';
+                const nounSingular = entityDataType?.nounSingular?.toLowerCase() ?? 'sample';
+                return (
+                    <Alert bsStyle="warning" className="left-margin">
+                        {dataCount} {dataCount > 1 ? nounPlural : nounSingular} will no longer be visible in this
+                        project. They won't be deleted and lineage relationships won't change.{' '}
+                    </Alert>
+                );
+            }
+
+            return null;
+        },
+        [uncheckedEntities, uncheckedEntitiesDB, dataCounts, entityDataType, dataTypes]
+    );
+
+    const headerLabel = useMemo(() => {
+        if (dataTypeLabel) return dataTypeLabel;
+
+        if (entityDataType?.typeNounSingular) return entityDataType.typeNounSingular + 's';
+
+        return null;
+    }, [dataTypeLabel, entityDataType]);
+
+    const getEntitiesSubList = useCallback(
+        (dataTypeEntities: DataTypeEntity[]): React.ReactNode => {
+            return (
+                <ul className="nav nav-stacked labkey-wizard-pills">
+                    {dataTypeEntities?.map((dataType, index) => {
+                        return (
+                            <li key={dataType.rowId} className="project-faceted__li">
+                                <div className="form-check">
+                                    <input
+                                        className="form-check-input filter-faceted__checkbox"
+                                        type="checkbox"
+                                        name={'field-value-' + index}
+                                        onChange={event => onChange(dataType.rowId, false, event.target.checked)}
+                                        checked={uncheckedEntities?.indexOf(dataType.rowId) < 0}
+                                        disabled={disabled}
+                                    />
+                                    <div
+                                        className="left-margin project-datatype-faceted__value"
+                                        onClick={() => onChange(dataType.rowId, true)}
+                                    >
+                                        {dataType.labelcolor && (
+                                            <ColorIcon
+                                                cls="label_color color-icon__circle-small"
+                                                value={dataType.labelcolor}
+                                            />
+                                        )}
+                                        {dataType.label}
+                                    </div>
+                                </div>
+                                {!!dataType.sublabel && (
+                                    <div className="help-block left-margin">{dataType.sublabel}</div>
```

Comment:
Seems to be a bit too much spacing above and below the text compared to the designs.

Reply:
good catch. updated